storage/buckets.rb

```diff
@@ -199,6 +199,133 @@ def delete_bucket project_id:, bucket_name:
   # [END delete_bucket]
 end
 
+def set_retention_policy project_id:, bucket_name:, retention_period:
+  # [START storage_set_retention_policy]
+  # project_id       = "Your Google Cloud project ID"
+  # bucket_name      = "Name of your Google Cloud Storage bucket"
+  # retention_period = "Object retention period defined in seconds"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project: project_id
```

Comment:
We are encouraging use of the `project_id` argument over the `project` argument. 

```
storage = Google::Cloud::Storage.new project_id: project_id
```

Reply:
acking

Reply:
We also encourage use of the `credentials` argument over `keyfile`, but I don't know if you are using that.